simcon_project/templates/conversation/conversation_step.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+{% load embed_video_tags %}
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0 shrink-to-fit=no">
+        <meta name="description" content="Created for Simulated Conversations Senior Capstone Project">
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+        <title>
+            {% block title %}Simulated Conversations | Conversation Step{% endblock %}
+        </title>
+
+        <!-- prevent line breaks when using form fields -->
+        <style type="text/css">
+            form {display:inline; margin:0px; padding:0px;}
+        </style>
+    </head>
+    <body>
+        <h1>Simulated Conversations</h1>
+        <h3>{{ ct.name }}</h3>
+        <h3 style="text-align:right;">{{ request.user }}</h3>
+        {% video ct_node.video_url 'small' %}
+        <p>{{ ct_node.description }}</p>
+        <form action="{{ ct_node.get_absolute_url }}" method="POST">
+            {% csrf_token %}
+            {{ choice_form }}
+            <button type="submit" class="btn btn-primary btn-sm">Next scene</button>
+        </form>
+        <!-- Optional JavaScript -->
+        <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+        <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+        <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+        <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
+    </body>
+    {% block extrahead %} {% endblock %}
+    <footer>
+        <p>Copyright 2020 Portland State University</p>
```

Comment:
do we have copy right?

Reply:
I actually don't know, I just followed the wireframe